sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -3,9 +3,17 @@
 # Licensed under the MIT License.
 # ------------------------------------
 import json
-from typing import Any, Union, List, Dict, Optional
+import sys
+from typing import Any, Dict, List, Optional, Union
+
+from azure.core.rest import HttpResponse
 from ._generated._serialization import Model
-from ._generated.models import KeyValue
+from ._generated.models import KeyValue, Snapshot as GeneratedSnapshot, ConfigurationSettingFilter
+
+if sys.version_info >= (3, 8):
+    from typing import Literal  # pylint: disable=no-name-in-module, ungrouped-imports
+else:
+    from typing_extensions import Literal  # type: ignore  # pylint: disable=ungrouped-imports
```

Comment:
We don't really need this if/else section.
You can just use:
`from typing_extensions import Literal` - and this will always work and it does this check for you.

Reply:
No, it'll have import issue without this check. Or do you mean it adds the check now?